archlinuxcn/cabal-fmt/PKGBUILD

```diff
@@ -0,0 +1,50 @@
+# Maintainer: berberman <hatsue@typed.icu>
+
+_hkgname=cabal-fmt
+pkgname=cabal-fmt
+pkgver=0.1.5
+pkgrel=1
+pkgdesc="Format .cabal files"
+url="https://github.com/phadej/cabal-fmt.git"
```

Comment:
This shouldn't be a git URL.

Reply:
Ah, the author does not provide the field `homepage` as usual, so arch-hs uses https://github.com/phadej/cabal-fmt/blob/dc1e63bb10ddafd6152fae8f7206c3b201b0f7ab/cabal-fmt.cabal#L23 as a fallback.